zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Schema.scala

```diff
@@ -113,13 +180,19 @@ trait Measurement {
 case class Schema(typeName: String,
                   dimension: Seq[Field],
                   measurement: Seq[Field],
-                  primaryKey: Seq[String],
-                  timeField: String
+                  primaryKey: Seq[Field],
+                  timeField: Option[TimeField]
                  ) {
 
   private val dimensionMap: Map[String, Field] = dimension.map(f => f.name -> f).toMap
   private val measurementMap: Map[String, Field] = measurement.map(f => f.name -> f).toMap
 
+  def apply(field: String): Field =
```

Comment:
I think `apply` is too succinct. I would suggest giving it a explicit name like `getField`